diana.navasardyan/repo/Lesson_3/exercise_05/exercise_05.py

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env python3
+
+number = int(input("Please enter a number  "))
+print("Enter a number",number, sep = ":")
```

Comment:
RW: This line should also be the user input line. There is no need to print the input number separately.